nipype/interfaces/ants/registration.py

```diff
@@ -379,6 +379,8 @@ class RegistrationInputSpec(ANTSCommandInputSpec):
         low=0.0, high=1.0, value=0.0, argstr='%s', usedefault=True, desc="The Lower quantile to clip image ranges")
 
     verbose = traits.Bool(argstr='-v', default=False)
+    profiling = traits.Bool(False, usedefault=True,
+                            desc='generate profiling output fields')
```

Comment:
There does not seem to be any performance penalty on collecting this information - is there a need for this input?